server/index.js

```diff
@@ -1,5 +1,7 @@
 const path = require("path");
 require('dotenv').config({ path: path.join(__dirname, '.env') });
+const env = require("../nuxt.config.js").env;
+
```

Comment:
Same with this file, I feel it should remain consistent and access `process`, unless there's some detriment to doing so.
```suggestion
```